### ISSUE TITLE
Refactoring build time configuration 

### ISSUE THREAD

**Issue by mssun** [apache/incubator-mesatee#134]
The current configuration mixes build time and runtime config. To separate them, here is what I can come up for build time configuration:

```toml
# Certificate to verify SGX enclaves
mr_signer_path = "cert/mr_signer"

# Service provider's root CA certificate to verify clients
sp_root_ca_cert_path = "cert/ca.crt"

# Intel Attestation Service root CA certificate to verify attestation report
ias_root_ca_cert_path = "cert/AttestationReportSigningCACert.pem"

# Auditors' public keys to verify their endorsement signatures
auditors_public_key_path = [
    "auditors/godzilla/godzilla.public.der",
    "auditors/optimus_prime/optimus_prime.public.der",
    "auditors/albus_dumbledore/albus_dumbledore.public.der"
]

# RPC max message size
rpc_max_message_size = 409600
```

This is also related to #131.

**Comment by mssun** [comment]
According to #135, `mr_signer_path` is useless. Here is an update of the build time config file:

```toml
# Service provider's root CA certificate to verify clients
sp_root_ca_cert = { path = "../cert/ca.crt" }

# Intel Attestation Service root CA certificate to verify attestation report
ias_root_ca_cert = { path = "../cert/AttestationReportSigningCACert.pem" }

# Auditors' public keys to verify their endorsement signatures
auditor_public_keys = [
    { path = "../auditors/godzilla/godzilla.public.der" },
    { path = "../auditors/optimus_prime/optimus_prime.public.der" },
    { path = "../auditors/albus_dumbledore/albus_dumbledore.public.der"},
]

# RPC max message size
rpc_max_message_size = 409600
```

The corresponding generated code will be like this:

```
format!(r#"
    pub struct BuildConfig<'a> {{
        pub sp_root_ca_cert: &'a [u8],
        pub ias_root_ca_cert: &'a [u8],
        pub auditor_public_keys: &'a [&'a [u8];{}],
        pub rpc_max_message_size: u64,
    }}

    pub static BUILD_CONFIG: BuildConfig<'static> = BuildConfig {{
        sp_root_ca_cert: {},
        ias_root_ca_cert: {},
        auditor_public_keys: {},
        rpc_max_message_size: {},
    }};"#,
        config.auditor_public_keys.len(),
        sp_root_ca_cert,
        ias_root_ca_cert,
        auditor_public_keys,
        config.rpc_max_message_size
    )
```

**Comment by mssun** [comment]
Fixed by #139.
